integration/src/main/java/scripts/testscripts/ResolveTdrDrs.java

```diff
@@ -6,38 +6,48 @@
 import bio.terra.testrunner.runner.TestScript;
 import bio.terra.testrunner.runner.config.TestUserSpecification;
 import java.util.Date;
+import java.util.List;
 import java.util.Map;
 import lombok.extern.slf4j.Slf4j;
 import org.junit.jupiter.api.Assertions;
 import scripts.utils.ClientTestUtils;
 
 @Slf4j
 public class ResolveTdrDrs extends TestScript {
-  private static String TDR_TEST_DRS_URI =
-      "drs://jade.datarepo-dev.broadinstitute.org/v1_93dc1e76-8f1c-4949-8f9b-07a087f3ce7b_8b07563a-542f-4b5c-9e00-e8fe6b1861de";
+  // All URIs pont to the same TDR hosted DRS object. Some URIs are aliases created in the TDR
+  // instance
+  private static final List<String> TDR_TEST_DRS_URIS =
+      List.of(
+          "drs://jade.datarepo-dev.broadinstitute.org/v1_93dc1e76-8f1c-4949-8f9b-07a087f3ce7b_8b07563a-542f-4b5c-9e00-e8fe6b1861de",
+          "drs://jade.datarepo-dev.broadinstitute.org/drshubfoo",
+          "drs://jade.datarepo-dev.broadinstitute.org/drshubfoo/bar");
```

Comment:
Did we want to add a url containing `%2F` here as a test? Also, we should talk about Pact testing between DRSHub and TDR!

Reply:
Ah good catch.  I'll add that